examples/benchmarks/bubblebench/simple-tree/.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*]
+indent_size = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[package.json]
+indent_size = 2
```

Comment:
personally I don't see the point of making package.json violate our normal indent conventions.

Most of our packages don't have a .editorconfig (we only have 4 of them repo wide), so it might be better to leave it out unless we specifically need it for this package for some reason.